templates/json/multitile/multitile.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "RENAME_ME",
+  "multitile": true,
+  "fg": [
+    "RENAME_ME_unconnected"
+  ],
+  "bg": [],
+  "additional_tiles": [
+    {
+      "id": "center",
+      "bg": [],
+      "fg": [
+        "RENAME_ME_center"
+      ]
+    },
+    {
+      "id": "corner",
+      "bg": [],
+      "fg": [
+        "RENAME_ME_corner_nw",
+        "RENAME_ME_corner_sw",
+        "RENAME_ME_corner_se",
+        "RENAME_ME_corner_ne"
+      ]
+    },
+    {
+      "id": "t_connection",
+      "bg": [],
+      "fg": [
+        "RENAME_ME_t_connection_n",
+        "RENAME_ME_t_connection_w",
+        "RENAME_ME_t_connection_s",
+        "RENAME_ME_t_connection_e"
+      ]
+    },
+    {
+      "id": "edge",
+      "bg": [],
+      "fg": [
+        "RENAME_ME_edge_ns",
+        "RENAME_ME_edge_ew"
+      ]
+    },
+    {
+      "id": "end_piece",
+      "bg": [],
+      "fg": [
+        "RENAME_ME_end_piece_n",
+        "RENAME_ME_end_piece_w",
+        "RENAME_ME_end_piece_s",
+        "RENAME_ME_end_piece_e"
+      ]
+    },
+    {
+      "bg": [],
+      "id": "unconnected",
+      "fg": [ "RENAME_ME_unconnected", "RENAME_ME_unconnected" ]
```

Comment:
I've copied it from `f_table`, is it supposed to be like this?

Reply:
If you mean two copies, yes, that tells the game that it should not rotate. Multitiles are assumed to rotate, so the game needs to be told that "all the rotations look the same" basically. It's worth adding a "//" comment to that effect maybe.